### ISSUE TITLE
[bug] 0.8.416 updater failing with Option 'KillPids' is unknown.

### ISSUE THREAD

**Issue by garfield69** [Jackett/Jackett#2140]
When using the *Check for updates* button to upgrade Jackett service on my win7 platform, the update fails. the updater.log shows
```
2017-11-14 06:23:36.5277 Info Jackett Updater v0.8.416 
2017-11-14 06:23:36.5902 Info Options "--Path" "C:\ProgramData\Jackett" "--Type" "jackettservice.exe" "--Args" " " "--KillPids" "2104" 
2017-11-14 06:23:36.7777 Error Jackett 0.8.416.0
Copyright ©  2015

ERROR(S):
  Option 'KillPids' is unknown.

  -p, --Path     Required. Install location

  -t, --Type     Install type

  -a, --Args     Launch arguments

  --norestart    Don't restart after update

  --killpids     PIDs which will be killed before (Windows) or after (Unix) the
                 update

  --help         Display this help screen.

  --version      Display version information.

 
2017-11-14 06:23:36.7777 Error Failed to process update arguments! 
```

**Comment by TRaSH-** [comment]
never been a issue before on windows

**Comment by kaso17** [comment]
this is broken since the command line parse change, already fixed via 7b80d3c07ed2f1ce1f69fcee405941f65100cf97 but seems like something else is broken (updater doesn't get started at all now).
@flightlevel please don't push any new releases, will fix it tomorrow.

**Comment by kaso17** [comment]
Fixed in version v0.8.423
